src/benchmarks/real-world/ImageSharp/General/Vectorization/DivFloat.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System.Numerics;
+using BenchmarkDotNet.Attributes;
+
+namespace SixLabors.ImageSharp.Benchmarks.General.Vectorization
+{
+    public class DivFloat
```

Comment:
I will stop commenting on these. `General/Vectorization` or better, the whole `General` folder should be dropped.